package/opt/hassbian/suites/monitor.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+function monitor-show-short-info {
+  echo "Setup for Monitor. Bluetooth-based passive presence detection."
+}
+
+function monitor-show-long-info {
+  echo "This script installs Monitor a script for Bluetooth-based passive presence detection."
+  echo "The Mosquitto suite will also be installed since it's a dependency for Monitor."
+  echo "This script also installs a service that can be enabled to run Monitor."
+}
+
+function monitor-show-copyright-info {
+  echo "Original concept by Landrash <https://github.com/landrash>."
+}
+
+function monitor-install-package {
+echo -n "Installing Mosquitto package: "
+
+echo "Checking the installation..."
+validation=$(pgrep -f mosquitto)
+if [ ! -z "${validation}" ]; then
+  echo "Existing Mosquitto install detected. Skipping install of Mosquitto."
+else
+  echo "Installing Mosquitto"
+sudo hassbian-config install mosquitto
+fi
+
+echo "Creating Monitor folder..."
+mkdir /opt/monitor || exit
+cd /opt || exit
+
+echo "Cloning Monitor git repository"
+git clone git://github.com/andrewjfreyer/monitor
+
+echo "Running interactive setup"
+cd /opt/monitor || exit
+sudo bash /opt/monitor/monitor.sh
```

Comment:
`sudo` is not needed inside the scripts :)

Reply:
Haha, didn't consider that..